Cargo.lock

```diff
@@ -2217,7 +2217,7 @@ dependencies = [
 [[package]]
 name = "kvproto"
 version = "0.0.2"
-source = "git+https://github.com/pingcap/kvproto.git?rev=511a3225c29f9272af83d28efed92eba835cd34f#511a3225c29f9272af83d28efed92eba835cd34f"
+source = "git+https://github.com/lhy1024/kvproto.git?rev=881f9799e333c2464c20b647da831ed5c9013d43#881f9799e333c2464c20b647da831ed5c9013d43"
```

Comment:
Why lhy1024 branch?

Reply:
After https://github.com/pingcap/kvproto/pull/638 merged, it will be replaced with pingcap branch.